nexus/src/app/instance.rs

```diff
@@ -1362,6 +1361,43 @@ impl super::Nexus {
         }
     }
 
+    async fn propolis_addr_for_instance(
```

Comment:
Could `propolis_client_for_instance` below use `propolis_addr_for_instance` to avoid duplicating code?

Reply:
For sure this is ripe for a refactor.
I want to be sure that we are actually using the correct address:port for the serial console and once I verify
that I'll refactor this to something better.

Reply:
Refactor done.